all-clouds/all-clouds-explanation/cloud-init-metapackages.rst

```diff
@@ -0,0 +1,21 @@
+Cloud-init metapackages
+=======================
+
+Starting in Ubuntu Oracular (24.10), `cloud-init <https://launchpad.net/ubuntu/+source/cloud-init>`_
+added cloud-specific metapackages to better express cloud-specific dependencies and reduce image size.
+
+* ``cloud-init-base`` contains the cloud-init code with the minimal set of dependencies
+
+* ``cloud-init`` is a metapackage that retains every previous dependency
+
+* ``cloud-init-cloud-sigma`` and ``cloud-init-smart-os`` are metapackages with an
```

Comment:
It would be helpful to have one or two examples of the cloud-specific metapackage names, along with a location where one can find the appropriate metapackage for their cloud, if such a location exists.

Reply:
`cloud-init-cloud-sigma` and `cloud-init-smart-os` are all the packages we plan for Oracular.

The [cloud-init source package](https://launchpad.net/ubuntu/+source/cloud-init), which is linked in the first sentence, is the canonical location to find them.

Reply:
Bonus: `cloud-init-azure` will come for Ubuntu PP.

Reply:
Note that the cloud-specific packages have not yet been implemented. This is being implemented in https://github.com/canonical/cloud-init/pull/5655.